### ISSUE TITLE
Improve user experience by displaying the appropriate command for setting variables on different platforms

### ISSUE THREAD

**Issue by F21** [hashicorp/vault#435]
When starting vault in dev mode, we get a message to export an environment variable:

```
The only step you need to take is to set the following
environment variables:

    export VAULT_ADDR='http://127.0.0.1:8200'

The unseal key and root token are reproduced below in case you
want to seal/unseal the Vault or play with authentication.
```

It would be nice if the command is changed to `set VAULT_ADDR=http://127.0.0.1:8200` when the server runs on windows.

**Comment by armon** [comment]
Tagged as enhancement, thanks!

**Comment by armon** [comment]
See: https://groups.google.com/forum/#!topic/vault-tool/xJMkrFEj_Xo

**Comment by F21** [comment]
This appears to be fixed in fbf21b5f2d3d94d1068b7de350b7ad06bd49593d.
